packages/flame/lib/src/camera/viewports/fixed_aspect_ratio_viewport.dart

```diff
@@ -21,10 +21,19 @@ class FixedAspectRatioViewport extends Viewport {
 
   Rect _clipRect = Rect.zero;
 
+  @override
+  void onLoad() {
+    final canvasSize = findGame()!.canvasSize;
+    _handleResize(canvasSize);
+  }
```

Comment:
Should `onLoad` be marked with `@mustCallSuper`. It now does something special that sub-classes might miss if they forget to call onLoad.

Reply:
True, good catch!